apps/wing/project-templates/wing/slack/main.w

```diff
@@ -0,0 +1,43 @@
+bring cloud;
+bring slack;
+
+/**
+ * In this example, we will create a slack application that gives us updates about
+ * a inbox bucket, sending slack messages when a new file is added. Additionally, the 
+ * app will allow us to mention it with the text "list inbox" to get a list of all the
+ * files in the inbox bucket.
+ */
+ 
+/// Since the slack bot will require a token as a secret please be sure 
```

Comment:
I believe `///` is intended for docstrings not for normal comments.

Let's fix this please because people use these templates as a style reference and we don't want this to be abused.